src/originalBudgetTrackingApp/setupHelpers.ts

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import moment from 'moment';
+import * as configManager from '@/originalBudgetTrackingApp/configManager/configManager';
+import { scrapeFinancialAccountsAndFetchTransactions } from '@/originalBudgetTrackingApp/import/importTransactions';
+import { getYnabAccountDetails } from './export/outputVendors/ynab/ynab';
+
+export async function printYnabAccountData() {
+  const ynabAccountData = await getYnabAccountDetails();
+  const companyIdToAccountNumbers = await getFinancialAccountNumbers();
+  console.log();
+  console.log();
+  console.log('------YNAB Budgets------');
+  console.log(ynabAccountData.budgets);
+  console.log();
+  console.log('------YNAB Accounts------');
+  console.log(ynabAccountData.accounts);
+  console.log();
+  console.log('------Financial institutions account numbers------');
+  console.log(companyIdToAccountNumbers);
+  console.log();
+}
```

Comment:
Why not in `ynab` module?

Reply:
I considered this. Since the setup of ynab requires matching account ids in ynab and the account ids in the financial and fetching all those ids is annoying, I wrote these to make it easier to do the setup. This is another flow of setup scripts (in addition to the flows of config and scrape) so I extracted it to a file of its own. 

I get your point that this is ynab specific so I will leave it in ynab and reexport it from here

Reply:
On second thought I think we should keep it separated in the setup helpers. As you can see this is very specific code that is only relevant to the setup flow. It uses ynab's `getYnabAccountDetails` as a building block but the rest is not related to the ynab api and I don't want to mix this code with the regular flow.

WDYT?

Reply:
If I understand you correctly, this is just an "external tool" to easily config the `ynab` at the first time (?).

If so, yes, let's keep it far from the library/module code. Even in a different folder, `tools` or something.

But to understand, if it not in the regular config+scrape flow, how can you run it? By running the code itself? How it published?

Reply:
Do not pay too much attention to where I said put it. Maybe the file is in the right place.

Reply:
At this point it is running the code itself. We could create a ui flow for setup that will do this behind the scenes and instead of printing it to the console will help you fill the config. But at this stage it is just a code tool.